content/dev/code/database_querying.md

```diff
@@ -135,15 +135,16 @@ $prod_name = $results->fields['products_name'];
 #### Multiple Row Responses
 If there are multiple rows returned, the most efficient way to iterate through the results is using a `foreach()` loop:
 
-```php
+```
```

Comment:
Was hugo reporting errors with this? 

Specifying the language is normal (and encouraged) in "github markdown". (That said, I realize hugo's translator doesn't match all features 1:1).

Reply:
The trouble is, with  `php` you get 80% width and that was wrapping the code in this case, making it hard to read.